motion/thruster_allocator/include/thruster_allocator/allocator_ros.hpp

```diff
@@ -0,0 +1,43 @@
+#ifndef VORTEX_ALLOCATOR_ALLOCATOR_ROS_HPP
+#define VORTEX_ALLOCATOR_ALLOCATOR_ROS_HPP
```

Comment:
use pragma once instead of include guards

Reply:
Disagree on this as it is discouraged by the CppCoreGuidelines and not ISO-compliant:
![image](https://github.com/vortexntnu/vortex-asv/assets/48532094/71b2272c-b695-4716-8e0b-6ae073c59bce)

https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#sf8-use-include-guards-for-all-h-files